src/pypit.py

```diff
@@ -180,27 +186,33 @@ def PYPIT(redname, progname=__file__, quick=False, ncpus=1, verbose=1, logname=N
         opt, arg = getopt.getopt(sys.argv[1:], 'hqc:v:', ['help',
```

Comment:
Like with the comment in arload, I think this should read:

`opt, arg = getopt.getopt(sys.argv[1:], 'hmqc:v:', ['help', 'use_masters',`